File: app/src/main/java/com/example/sarahn/waterreminderapp/classes/ClimateValueChangedListener.java
package com.example.sarahn.waterreminderapp.classes;

import android.widget.NumberPicker;

/**
 * Created by SarahN on 6/16/2017.
 */
public class ClimateValueChangedListener implements NumberPicker.OnValueChangeListener {
    @Override
    public void onValueChange(NumberPicker picker, int oldVal, int newVal) {
        if(ClimateBottomSheet.values.length == 0){
            return;
        }else{
            ClimateBottomSheet.tv.setText(ClimateBottomSheet.values[newVal-1]);
        }

    }
}

File: app/src/main/java/com/example/sarahn/waterreminderapp/classes/ClimateValueChangedListener.kt
package com.example.sarahn.waterreminderapp.classes

import android.widget.NumberPicker
import android.widget.NumberPicker.OnValueChangeListener

/**
 * Created by SarahN on 6/16/2017.
 */
class ClimateValueChangedListener : OnValueChangeListener {
    override fun onValueChange(picker: NumberPicker, oldVal: Int, newVal: Int) {
        if (ClimateBottomSheet.values?.size == 0) {
            return
        } else {
            ClimateBottomSheet.tv!!.text = ClimateBottomSheet.values?.get(newVal - 1) ?: ""
        }
    }
}